app/src/lib/dispatcher/error-handlers.ts

```diff
@@ -281,6 +281,14 @@ export async function mergeConflictHandler(
     return error
   }
 
+  if (e.metadata.command) {
```

Comment:
Explicit check for `e.metadata.command`. Additionally, we could `return` to avoid nesting `if` statements.

Reply:
Try out f1ba9308e and see how that feels